<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginCard</class>
 <widget class="QFrame" name="PluginCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string>QFrame#PluginCard {
    background-color: palette(base);
    border: 1px solid palette(mid);
    border-radius: 5px;
    padding: 10px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Shadow::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="styleSheet">
        <string>font-size: 16px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Plugin Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="actionBtn">
       <property name="text">
        <string>Install</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="dependenciesFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="depsLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="depsHeaderLabel">
        <property name="styleSheet">
         <string>font-weight: bold; margin-top: 5px;</string>
        </property>
        <property name="text">
         <string>Dependencies:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="depsContainer" native="true">
        <layout class="QVBoxLayout" name="depsContainerLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
